.changelog

```diff
@@ -4,8 +4,11 @@
 # - Fixed error in Bat section of Upgrade Guide (bsc#1234567)
 # For guidelines: https://en.opensuse.org/openSUSE:Creating_a_changes_file_(RPM)
 
+- Added support for SUSE Linux Enterprise Micro 5.5 and openSUSE 
+  Leap Micro 5.5 clients to Installation and Upgrade Guide, and to
+  Configuration Guide
```

Comment:
Maybe,mentioning here that ppc64le is not supported, and thus removed from the matrix?

Reply:
Not needed, it was a mistake in one of the matrices. This is a correction of faulty data, not the amendment of it.